mlx/io/gguf_quants.cpp

```diff
@@ -9,15 +9,18 @@
 namespace mlx::core {
 
 void unpack_32_4(uint8_t* data, int8_t* dst) {
-  for (int64_t j = 0; j < 16; ++j) {
+  for (int j = 0; j < 16; ++j) {
+    dst[j] = 0;
+  }
```

Comment:
Out of curiosity, why not `std::fill_n(dst, 16, 0);`?

Possible answers
a) it fits much better with the rest of the function
b) why include a header instead of 3 lines
c) compiles faster
d) all of the above 😅

Reply:
e) didn't think of it